vanilla-javascript-demo/package.json

```diff
@@ -0,0 +1,17 @@
+{
+  "name": "vanilla-javascript-demo",
+  "version": "1.0.0",
+  "private": true,
+  "description": "Example of building ChemDraw JS application with JavaScript",
+  "scripts": {
+    "start": "serve",
+    "preinstall": "rm -rf asset",
+    "postinstall": "mkdir asset && cp ../dependencies/chemdraw-js-license.xml $INIT_CWD/asset && cp -r node_modules/@pki-chemistry/chemdraw-js/dist/chemdrawweb $INIT_CWD/asset"
```

Comment:
You can't use cp as it's not cross platform. You will need to install one of the many packages that provide a copy cli tool and use that instead. You shouldn't use mkdir either, I don't think that's needed anyway as the directory should be created by the copy.

You also seem to be using some kind of environment variable for current directory, but this is unnecessary, you should just assume you are in the directory of the package.json.

Also, your assets should be in a folder called "assets" not "asset".